elements/map/src/controls/controls.css

```diff
@@ -26,3 +26,9 @@
   width: 100%;
   height: 100%;
 }
+
+.disaibled-button {
+  opacity: 0.8;
+  filter: contrast(0.5);
+  cursor: not-allowed;
```

Comment:
It looks a bit strange with the reduced opacity and the button being so dark; how about we
- make it brighter
- disable pointer-events? 
so something like:
```
{
  filter: brightness(1.7);
  pointer-events: none;
}
```